aws_lambda_builders/workflows/dotnet_clipackage/dotnetcli.py

```diff
@@ -49,14 +50,16 @@ def run(self, args, cwd=None):
 
         LOG.debug("executing dotnet: %s", invoke_dotnet)
 
+        # DotNet output is in system locale dependent encoding
```

Comment:
Is there a doc page which cites this? can we add that here for future reference?

Reply:
Added doc.